db/tests/run-db-tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env  bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# This script is supposed to be run from a context in which node is available. Typically
+# you run this via "make dbtest" and it will then spin up the docker container for the
+# test database, initialize it and then run the tests
+
+: "${GRAPHER_TEST_DB_USER:?Need to set GRAPHER_TEST_DB_USER non-empty}"
+: "${GRAPHER_TEST_DB_PASS:?Need to set GRAPHER_TEST_DB_PASS non-empty}"
+: "${GRAPHER_TEST_DB_NAME:?Need to set GRAPHER_TEST_DB_NAME non-empty}"
+
+echo '==> Starting & initializing Mysql instance for testing'
+
+docker-compose -f docker-compose.dbtests.yml up -d
+
+./devTools/docker/wait-for-tests-mysql.sh
+
+sleep 3s # This is to give the db init script time to create the user, database and tables
```

Comment:
Not sure if this is a WSL thing, but `sleep 3s` doesn't work on Mac. `sleep 3` does

Reply:
```suggestion
sleep 3 # This is to give the db init script time to create the user, database and tables
```

Yes, this also broke for me. On Linux, the `s` suffix is optional, it defaults to seconds anyway. On MacOS it does not accept the suffix and only accepts seconds.

Reply:
Actually, even after getting it working, I was getting tons of spurious failures due to this sleep. Added a PR on top of this branch with a suggested fix: https://github.com/owid/owid-grapher/pull/1890